### PR TITLE
fix: updated readme file

### LINKED ISSUE #14: [Eco Pass KIT v2.0.0] Update main readme
# What needs to be done?

- [ ] Add logos to the main readme
- [ ] Add Documentation of Interest (in reference implementation)
- [ ] Add readme headers like the reference implementation

### PR DESCRIPTION
**Pull request by dsrparracho**
# Why we create this PR?
 
The main README is outdated and needs to be updated
 
# What we want to achieve with this PR?
 
Update the main README file.
 
# What is new?

##Added
The Eco Pass KIT icon into /resources/img folder

##Updated
The README file.

Closes  #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +44,37 @@ Provide a detailed description and offer tools to implement product passports.
...
+| [Adoption View](/docs/page-adoption-view.md)                     | Technical Guide - Deployment in ArgoCD Hotel Budapest (integration environment)                                                                             |
+| [Development View](/docs/page-software-development-view.md)                                      | Overview on general docker commands                                                                                                                         |
+| [Operation View](/docs/page-software-operation-view.md)                     | This guide describes how to setup a keycloak instance in local docker container and import existing realm.json file.                                        |                                                                                                       |
+
+
+
+<!-- MARKDOWN LINKS & IMAGES -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+[contributors-shield]: https://img.shields.io/github/contributors/eclipse-tractusx/digital-product-pass.svg?style=for-the-badge
```
Comment (matbmoser) [inline README.md]: We are still pointing to digital product pass links, we need to add this badges from the eco pass kit

**Review by matbmoser — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+![Eco Pass Kit Pictotogram](/docs/resources/img/EcoPassKIT_pictogram.png)
+
+<h1 style="display:flex; align-items: center;">&nbsp;&nbsp;Eco Pass KIT</h1>
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Apache 2.0 License][license-shield]][license-url]
+[![Latest Release][release-shield]][release-url]
 
 This is the repository of ECO-Pass KIT. In this repository we collect all the information needed to use the this KIT.
```
Comment (maxim-bo) [inline README.md]: "repository of **the** EcoPass Kit"
information needed to use **"the" or "this"** KIT

File: README.md [hunk 3]
```diff
@@ -12,3 +44,37 @@ Provide a detailed description and offer tools to implement product passports.
 
 We have an reference implementation which is the Digital Product Pass.
 You find the repository [here](https://github.com/eclipse-tractusx/digital-product-pass)
```
Comment (maxim-bo) [inline README.md]: You **can** find the repository

File: README.md [hunk 4]
```diff
@@ -12,3 +44,37 @@ Provide a detailed description and offer tools to implement product passports.
...
+
+### Documentation
+
+To get started you can have a look into our documentation:
+
+| Name                                                                      | Description                                                                                                                                                 |
+| ----------------------------------------------------------------          |-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Arc42](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/arc42/Arc42.md)                                             | Main Architecture Document (Arc42) of Digital Product Pass Application                                                                                      |
```
Comment (maxim-bo) [inline README.md]: (Arc42) of **the** Digital Product Pass Application

File: README.md [hunk 5]
```diff
@@ -12,3 +44,37 @@ Provide a detailed description and offer tools to implement product passports.
...
+| [Administration Guide](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/admin%20guide/Admin_Guide.md)                  | Administration Guide explaining the infrastructure and how to configure the application                                                                     |
+| [Data Retrieval Guide](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/data%20retrieval%20guide/DataRetrievalGuide.md)          | Backend documentation Product Passport App                                                                                                                  |
```
Comment (maxim-bo) [inline README.md]: documentation **of the** Product ...

**Review by maxim-bo — CHANGES_REQUESTED** [review]
Again, some minor typos and so on. Other than that it seems good to me (y)

File: README.md (hunk 5 above)
Comment (matbmoser) [reply README.md]: Fixed!

File: README.md (hunk 4 above)
Comment (matbmoser) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (matbmoser) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (matbmoser) [reply README.md]: fixed

**Review by matbmoser — APPROVED** [review]

**Pull request merged**